### PR TITLE
LPS-198417 [Marketplace] Adjustments to the email template

### PR DESCRIPTION
**Pull request by eliasantosf**
Task: [LPS-198417](https://issues.liferay.com/browse/LPS-198417)

### CODE REVIEW

**Comment by eliasantosf** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/next-steps-template-free/en-US.html [hunk 1]
```diff
@@ -129,8 +129,8 @@ <h2 style=" font-size: 30px; font-weight: bolder; margin-top: 2.5rem; margin-bot
 						</div>
 
 						<div style=" padding: 10px; text-align: center; margin: 2.5rem; ">
-							<a href="[%GETAPPINFORMATION_DASHBOARDLINK%]" style=" background-color: #ffffff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #000; font-weight: bold; border: #000 1px solid; margin-right: 1rem; ">Go to Dashboard</a>
-							<a href="https://console.marketplacedemo.liferay.sh/projects" style=" background-color: #0b5fff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #ffffff; font-weight: bold; ">Go to Cloud Console</a>
+							<a href="[%GETAPPINFORMATION_DASHBOARDLINK%]" style=" background-color: #ffffff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #000; font-weight: bold; border: #000 1px solid; margin-right: 1rem; " hidden>Go to Dashboard</a>
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/next-steps-template-free/en-US.html:132]: Why are you using the hidden attirube in this link?

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/next-steps-template-free/en-US.html [hunk 2]
```diff
@@ -129,8 +129,8 @@ <h2 style=" font-size: 30px; font-weight: bolder; margin-top: 2.5rem; margin-bot
...
+							<a href="https://console.marketplacedemo.liferay.sh/projects" style=" background-color: #0b5fff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #ffffff; font-weight: bold; ">Continue Configuration</a>
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/next-steps-template-free/en-US.html:133]: Please, remove the spaces in this line.
```suggestion
							<a href="https://console.marketplacedemo.liferay.sh/projects" style=" background-color: #0b5fff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #ffffff; font-weight: bold;">Continue Configuration</a>
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/next-steps-template-paid-or-trial/en-US.html [hunk 3]
```diff
@@ -133,8 +133,8 @@ <h2 style=" font-size: 30px; font-weight: bolder; margin-top: 2.5rem; margin-bot
 						</div>
 
 						<div style=" padding: 10px; text-align: center; margin: 2.5rem; ">
-							<a href="[%GETAPPINFORMATION_DASHBOARDLINK%]" style=" background-color: #ffffff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #000; font-weight: bold; border: #000 1px solid; margin-right: 1rem; ">Go to Dashboard</a>
-							<a href="https://console.marketplacedemo.liferay.sh/projects" style=" background-color: #0b5fff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #ffffff; font-weight: bold; ">Go to Cloud Console</a>
+							<a href="[%GETAPPINFORMATION_DASHBOARDLINK%]" style=" background-color: #ffffff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #000; font-weight: bold; border: #000 1px solid; margin-right: 1rem; " hidden>Go to Dashboard</a>
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/next-steps-template-paid-or-trial/en-US.html:136]: Why are you using hidden attribute in this link?

Please, remove the spaces here.
```suggestion
							<a href="[%GETAPPINFORMATION_DASHBOARDLINK%]" style=" background-color: #ffffff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #000; font-weight: bold; border: #000 1px solid; margin-right: 1rem;" hidden>Go to Dashboard</a>
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/next-steps-template-paid-or-trial/en-US.html [hunk 4]
```diff
@@ -133,8 +133,8 @@ <h2 style=" font-size: 30px; font-weight: bolder; margin-top: 2.5rem; margin-bot
...
+							<a href="https://console.marketplacedemo.liferay.sh/projects" style=" background-color: #0b5fff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #ffffff; font-weight: bold; ">Continue Configuration</a>
```
Comment (silasbsi) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/next-steps-template-paid-or-trial/en-US.html:137]: Please, remove the spaces here.
```suggestion
							<a href="https://console.marketplacedemo.liferay.sh/projects" style=" background-color: #0b5fff; padding: 0.5rem 1.5rem; border-radius: 8px; width: 10rem; text-decoration: none; height: 2.5rem; color: #ffffff; font-weight: bold;">Continue Configuration</a>
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added some questions and suggestions.

**Comment by silasbsi** [comment]
ci:forward:force

**Pull request closed without merging**